snapcraft/pack.py

```diff
@@ -90,6 +90,64 @@ def _get_filename(
     return None
 
 
+def _pack(command: List[Union[str, Path]]) -> str:
+    """Pack a directory with `snap pack` as a snap or component.
+
+    :param command: `snap pack` command to execute
+
+    :returns: The filename of the packed snap or component.
+
+    :raises SnapcraftError: If the directory cannot be packed.
+    """
+    emit.debug(f"Pack command: {command}")
+
+    try:
+        proc = subprocess.run(
+            command, capture_output=True, check=True, universal_newlines=True
+        )
+    except subprocess.CalledProcessError as err:
+        msg = f"{err!s}"
```

Comment:
This could be str(err) (but I see that other parts of the code are using this pattern).

Reply:
Good point.  This was previously existing code, but I made this change.